enterprise/server/configure.md

```diff
@@ -101,12 +101,30 @@ The `*` wildcard character matches any value. Wildcards are supported in the fol
 
 ````
 ephox {
-	allowed-origins {
-		origins = [ "http://myserver:*", "*://myotherserver.example.com", "*://*.mydomain.example.com:*"]
-	}
+  allowed-origins {
+    origins = [ "http://myserver:*", "*://myotherserver.example.com", "*://*.mydomain.example.com:*"]
+  }
 }
 ````
 
+#### `allowed-origins.same-origin` (optional)
+
+{{site.requires_jsscwar_230v}}
+
+The `allowed-origins` `same-origin` option can be used to block all cross-origin requests. This option is set to `false` by default.
+
+Setting the `same-origin` setting to `true` will block all HTTP `OPTIONS` requests and allow all other HTTP methods. When set to `true`, all `origins` specified in `allowed-origins` will be ignored.
```

Comment:
Do we need to include the purpose of why someone would want this setting?

Reply:
added a why: https://github.com/tinymce/tinymce-docs/pull/1518/commits/771c52fe3bd2e2c75e93dc057f9ccc6bf08a7dc0